Detectors/ZDC/macro/CreateBaselineCalib.C

```diff
@@ -0,0 +1,86 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#if !defined(__CLING__) || defined(__ROOTCLING__)
+
+#include "Framework/Logger.h"
+#include "CCDB/CcdbApi.h"
+#include "ZDCBase/Constants.h"
+#include "ZDCReconstruction/BaselineParam.h"
+#include <string>
+#include <TFile.h>
+#include <map>
+
+#endif
+
+using namespace o2::zdc;
+using namespace std;
+
+void CreateBaselineCalib(long tmin = 0, long tmax = -1, std::string ccdbHost = "")
+{
+
+  BaselineParam conf;
```

Comment:
it would be useful to have an option to simply write a local file.

Reply:
I tried to introduce some helper functions to remove repeated code in all macros. Everything works in compiled mode but the helper functions are not found in interpreted mode. They are defined in the ZDC namespace. Any suggestion?

 root -q /home/cortese/alice/O2/Detectors/ZDC/macro/CreateBaselineCalib.C'(0,-1,"root")'

Processing /home/cortese/alice/O2/Detectors/ZDC/macro/CreateBaselineCalib.C(0,-1,"root")...
In file included from input_line_9:1:
/home/cortese/alice/O2/Detectors/ZDC/macro/CreateBaselineCalib.C:72:36: error: no member named 'ccdbShortcuts' in namespace 'o2::zdc'
  std::string ccdb_host = o2::zdc::ccdbShortcuts(ccdbHost, conf.Class_Name(), o2::zdc::CCDBPathBaselineCalib);
                          ~~~~~~~~~^
/home/cortese/alice/O2/Detectors/ZDC/macro/CreateBaselineCalib.C:74:16: error: no member named 'endsWith' in namespace 'o2::zdc'
  if (o2::zdc::endsWith(ccdb_host, ".root")) {